js/app/modules/contentGroup/mediaLightbox.js

```diff
@@ -56,6 +56,7 @@ const MediaLightbox = new Module.Class({
         if (this.lightbox_widget instanceof EosKnowledgePrivate.MediaBin) {
             this.lightbox_widget.stop();
         }
+        this.drop_submodule(this.lightbox_widget);
```

Comment:
Not really, in the previous commit I added the dropped() method not drop_submodule()